pyproject.toml

```diff
@@ -102,9 +103,15 @@ exclude = '''
 
 [tool.pylint.master]
 # Include the pylint_django plugin to avoid spurious warnings about Django patterns
-load-plugins="pylint_django"
+load-plugins="pylint_django,pylint_nautobot"
 ignore=".venv"
 
+[tool.pylint-nautobot]
+supported_nautobot_versions = [
+  "1",  # Supporting 1.x.y
```

Comment:
I think the goal with this branch is not to support version 1, right?

Reply:
Was just left there cause it wasn't hurting anything. I've removed it now though.